php/src/Docker/DockerActionManager.php

```diff
@@ -842,26 +840,26 @@ public function sendNotification(Container $container, string $subject, string $
         }
     }
 
-    private function DisconnectContainerFromBridgeNetwork(string $id) : void
-    {
-
-        $url = $this->BuildApiUrl(
-            sprintf('networks/%s/disconnect', 'bridge')
-        );
-
-        try {
-            $this->guzzleClient->request(
-                'POST',
-                $url,
-                [
-                    'json' => [
-                        'container' => $id,
-                    ],
-                ]
-            );
-        } catch (RequestException $e) {
-        }
-    }
+    // private function DisconnectContainerFromBridgeNetwork(string $id) : void
+    // {
+
+    //     $url = $this->BuildApiUrl(
+    //         sprintf('networks/%s/disconnect', 'bridge')
+    //     );
+
+    //     try {
+    //         $this->guzzleClient->request(
+    //             'POST',
+    //             $url,
+    //             [
+    //                 'json' => [
+    //                     'container' => $id,
+    //                 ],
+    //             ]
+    //         );
+    //     } catch (RequestException $e) {
+    //     }
+    // }
```

Comment:
Remove dead code instead of commenting out?
You can get it in git history if needed.

Reply:
hm, okay